modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +202,298 @@
 	damage = 60
 */
 
+/// The Domestic.
+/obj/item/gun/ballistic/automatic/pistol/security
+	name = "Domestic"
```

Comment:
```suggestion
	name = "'Domestic' pistol"
```
Let's make sure people know what it actually is when you pull it out and get the big ol' **X grabs the Y!** that everyone can see

Reply:
ye